### DISCUSSION TITLE
Support for Rank Math soon?

### DISCUSSION THREAD

**Discussion by ioancruso** [GatoGraphQL/GatoGraphQL#2505] [General]
HI! I started using the plugin to build a headless frontend for my website and I wonder if there will be any support for Rank Math soon? Or there is any way I can get the meta descriptions from the posts?  Thanks in advance

**Comment by leoloso** [comment]
I do plan to create an integration for Rank Math, but it won't be anytime soon.

You can create a custom Field Resolver to retrieve the meta descriptions. Once `v1.0` of Gato GraphQL is published on the wp.org plugin directory (already submitted for approval; ETA: some 2 months), I'll create a starter project to make it easier for anyone to create extensions.

If you can't wait, you can create a temporary hack already, modifying the code inside plugin. Check the source code for some Field Resolver for posts, and see if you can replicate the logic.

**Reply by ioancruso** [reply]
Thanks! I will have a look.
